backend/graminsta/post/utils.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+"""utils.py"""
+from django.contrib.auth import models as auth_models
+from .models import FollowRelationship
+
+
+def get_people_user_follows(user):
+    """
+    Returns a QuerySet representing the users that the given user follows.
+    """
+    ids = FollowRelationship.objects.filter(from_user=user)\
+        .values_list('to_user', flat=True)
```

Comment:
I think internally, Django is saving `to_user_id` in the db column. Not sure if there is side effects if you use `to_user` here, prefer to change it to `to_user_id` to make it explicit instead.